step-templates/iis-website-update-property.json

```diff
@@ -5,7 +5,7 @@
   "ActionType": "Octopus.Script",
   "Version": 5,

```

Comment:
The version needs to be incremented, otherwise the update won't be picked up in Octopus.